Dockerfile.backend

```diff
@@ -0,0 +1,20 @@
+# HACK: I would use multistage build, but travis has old version of docker
```

Comment:
You can install a newer Docker version
https://docs.travis-ci.com/user/docker/#Installing-a-newer-Docker-version